.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -12,53 +12,88 @@
 name: AGW Test LTE Integration With Make Containerized Build
 
 on:
-  workflow_dispatch: null
```

Comment:
Doesn't it make sense to be able to trigger this manually?

Reply:
No, I do not think that this makes sense. Essentially, everything testable should be tested by default. Any manual trigger would only test a "latest" configuration (and changing this is not worth the effort imho). Moreover this would give the option to execute it from any branch, which I find rather undesireable and might even hold security implications.

Reply:
If at all, this should be triggered via an event on the upstream job.

Reply:
A similar workflow that download and tests a prebuilt artifact is the [LTE Integtest against the .deb package](https://github.com/magma/magma/blob/master/.github/workflows/lte-integ-test-magma-deb.yml), and there we allow manual execution. One advantage that I see is that you can trigger the integtest without doing a full rebuild, which is much faster. A usecase that comes to mind is retriggering the workflow after it failed due to an infrastructure hiccup.

Reply:
Readded. :heavy_check_mark: